data/859/074/31/85907431.geojson

```diff
@@ -4,21 +4,20 @@
   "properties": {
     "edtf:cessation":"uuuu",
     "edtf:inception":"uuuu",
-    "geom:area":0.0,
+    "geom:area":0.000482,
     "geom:area_square_m":0.0,
-    "geom:bbox":"46.810475,24.675125,46.810475,24.675125",
-    "geom:latitude":24.675125,
-    "geom:longitude":46.810475,
+    "geom:bbox":"46.79613,24.66791,46.827154,24.695813",
+    "geom:latitude":24.681904,
+    "geom:longitude":46.811557,
     "gn:id":"",
```

Comment:
This isn't from the PR, but odd we have a `"gn:id":""` here.

Reply:
Agreed.. should we remove these properties (when empty) in this PR?

Reply:
It's probably affecting more tiles, so file a general cleanup issue?

Reply:
Tracking as a subtask in https://github.com/whosonfirst-data/whosonfirst-data/issues/866.